scipy/stats/stats.py

```diff
@@ -1114,6 +1120,8 @@ def variation(a, axis=0, nan_policy='propagate'):
           * 'propagate': returns nan
           * 'raise': throws an error
           * 'omit': performs the calculations ignoring nan values
```

Comment:
Another tweak of the markup:
```suggestion
        * 'propagate': returns nan
        * 'raise': throws an error
        * 'omit': performs the calculations ignoring nan values

```